src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -665,19 +740,36 @@ sub-<label>/[ses-<label>/]             # MultipartID
 
 ### Other RECOMMENDED metadata
 
+#### Phase-encoding information (common metadata fields)
+
 The `PhaseEncodingDirection` and `TotalReadoutTime` metadata
 fields are RECOMMENDED to enable the correction of geometrical distortions
 with [fieldmap information](#fieldmap-data).
 See [Common metadata fields](#common-metadata-fields) for a list of
 additional terms that can be included in the corresponding JSON file.
 
+#### Pulse gradient timing information
+
+Although the *b*-value is the most common way to convey gradient strength, it does not
+unambiguously depict the sampled coordinate in *q*-space.
+The relationship between each *b*-vector and its corresponding *q*-space coordinate
+can be calculated with the `BigDelta` and `SmallDelta` metadata fields in the corresponding
+JSON file.
+
+| Field name     | Definition                                                                                                                                                                                                                                                                |
+| :------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| SmallDelta     | RECOMMENDED. Acquisition pulse separation time (in seconds). For Siemens, this is in the shadow header (0029,1120) under `sSpecPara.dSpecLipidSupprBandwidth` in milliseconds. This value is necessary for estimating ensemble average propagators in physical units.     |
+| BigDelta       | RECOMMENDED. Acquisition pulse separation time (in seconds). For Siemens, this is in the shadow header (0029,1120) under `sSpecPara.dSpecLipidSupprDeltaPos` in milliseconds. This value is also necessary for estimating ensemble average propagators in physical units. |
+
 JSON example:
 
 ```JSON
 {
   "PhaseEncodingDirection": "j-",
   "TotalReadoutTime": 0.095,
-  "B0FieldSource": ["phasediff_fmap0", "pepolar_fmap0"]
+  "B0FieldSource": ["phasediff_fmap0", "pepolar_fmap0"],
+  "BigDelta": 0.005,
+  "SmallDelta": 0.0005
```

Comment:
-   Given that these appear in a general sidecar metadata space rather than a DWI-specific context, the names "`BigDelta`" and "`SmallDelta`" may not be specific enough.

-   If revisiting, it would be worth considering whether it is worthwhile adding these, or instead deferring more advanced diffusion encoding information entirely to the aDWI extension.